CONTRIBUTING.md

```diff
@@ -77,6 +77,36 @@ Before you can build the Cisco Webex JS SDK, you will need the following depende
 >
 > **We suggest using [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about) (WSL) for the best experience on Windows.** We've seen multiple tooling/package related errors when trying to build on a regular Windows environment. The build errors are a result of third-party node_modules that we do not maintain. These issues are not seen when using WSL.
 
+### Building the SDK
+
+Ensure that you have followed all the steps outlined in [Build Dependencies](#build-dependencies).
+
+Fork the [webex-js-sdk](https://github.com/webex/webex-js-sdk/) repository and `git clone` your fork:
+
+```bash
+git clone https://github.com/your-username/webex-js-sdk.git
+```
+
+Install tooling dependencies with:
+
+```bash
+npm install
```

Comment:
I would add changing node versions first, before installing dependencies.
I know it's called out in the "Build dependencies section" but in case anyway skipped/skimmed it it's a simple sanity check.

```suggestion
nvm use
npm install
```

Reply:
Done!